### PR TITLE
Redesigned calendar

### PR DESCRIPTION
**Pull request by sianis**
+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: UNRELEASED.md [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+  - Added new style value `calendarDateSelectedSameDayBackgroundColor` to set the secondary circle color
+  - Increased `rowheight`
+  - Added new style parameter to London and Doha themes
+  - Default font weight is `Emphasized`
+  - Removed today marking circle
+  - `Colored` calendar shows color around days instead of below them
+
+**Fixed:**
+- `MonthView`:
+  - Range rectangular canvas drawable height is calculated from circle size
```
[the remark this answers is not in the document]
Comment (sianis) [reply UNRELEASED.md]: This is the how. With original size the old was fine, but new values broke  it.

[the remark this answers is not in the document]
Comment (antalindisguise) [reply UNRELEASED.md]: I would imagine we would write here what was fixed and not how

[the remark this answers is not in the document]
Comment (sianis) [reply UNRELEASED.md]: Changed

File: Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt [hunk 2]
```diff
@@ -416,16 +364,15 @@ internal class MonthView @JvmOverloads constructor(
     day: LocalDate,
     range: CalendarRange,
     paddingX: Int,
-    paddingY: Int,
     x: Int,
     y: Int
   ) {
-    rangeBackPaint.color = if (isColoredCalendar()) rangeColorColored else rangeColorNonColored
+    rangeBackPaint.color = rangeColorNonColored
```
Comment (bvitaliyg) [inline Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt]: It seems like rangeBackPaint always have `rangeColorNonColored` colour. This assignments may be removed now.

**Pull request closed without merging**